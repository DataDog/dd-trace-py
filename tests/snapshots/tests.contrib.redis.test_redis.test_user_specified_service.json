[[
  {
    "name": "redis.command",
    "service": "redis",
    "resource": "GET cheese",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "redis",
    "meta": {
      "_dd.base_service": "mysvc",
      "_dd.p.dm": "-0",
      "component": "redis",
      "db.system": "redis",
      "language": "python",
      "out.host": "localhost",
      "redis.raw_command": "GET cheese",
      "runtime-id": "9d4dd102c4394715976611e15b961233",
      "span.kind": "client"
    },
    "metrics": {
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "db.row_count": 0,
      "network.destination.port": 6379,
      "out.redis_db": 0,
      "process_id": 20056,
      "redis.args_length": 2
    },
    "duration": 439500,
    "start": 1692651822941153668
  }]]
