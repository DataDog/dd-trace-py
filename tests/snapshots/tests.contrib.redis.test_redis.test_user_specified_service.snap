[[{"name" "redis.command"
   "service" "redis"
   "resource" "GET cheese"
   "type" "redis"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1596062189619805000
   "duration" 2395000
   "meta" {"runtime-id" "aa72b38bf1554ce49a747687494a4751"
           "redis.raw_command" "GET cheese"
           "out.host" "localhost"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 76425
              "out.port" 6379
              "out.redis_db" 0
              "redis.args_length" 2}}]]
